.github/release_issue_template.md

```diff
@@ -0,0 +1,20 @@
+# Documentation Release Checklist
+
+Use this issue template for documentation releases.
+
+## Packaging Day:
+
+- [ ] Ensure all relevant PRs are merged (and backported where required)
```

Comment:
How?  Is it enough to check https://github.com/uyuni-project/uyuni-docs/pulls?  Do we also have to check the board?